modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -16,7 +16,7 @@ This procedure uses a YAML file to demonstrate editing the configuration and app
 
 .Procedure
 
-. Create or edit a virtual machine configuration that you want to connect to the bridge network. Add the bridge interface to the `spec.template.spec.domain.devices.interfaces` list and the network attachment definition to the `spec.template.spec.networks` list. The name of the `interfaces` entry must be the same as the `networks` entry. The following example adds a bridge interface called `bridge-net` that connects to the `a-bridge-network` network attachment definition:
+. Create or edit a virtual machine configuration that you want to connect to the bridge network. Add the bridge interface to the `spec.template.spec.domain.devices.interfaces` list and the network attachment definition to the `spec.template.spec.networks` list. The name of the `interfaces` entry must be the same as the `networks` entry. This example adds a bridge interface called `bridge-net` that connects to the `a-bridge-network` network attachment definition:
```

Comment:
You can remove "The name of the `interfaces` entry must be the same as the `networks` entry" from this line. The callout says this very clearly.

"virtual machine configuration that you want to connect to the bridge network" - Small nit. This sounds like you're connecting the configuration to the bridge network. Might be clearer if you change it to "configuration of a virtual machine that you want to connect to the bridge network".

Reply:
Done. I also made "Add the bridge interface..." a separate step in the procedure. It seemed to make more sense that way.